drivers/ethernet/eth_stm32_hal.c

```diff
@@ -777,6 +777,7 @@ static void eth_iface_init(struct net_if *iface)
 {
```

Comment:
The commit subject could be made shorter if needed, so here it would be enough to write `drivers: ethernet: stm32: Enable IRQ ....`, usually we do not put file name to the subject.

Reply:
Thanks. Done